mula/scheduler/connectors/services/services.py

```diff
@@ -38,7 +38,13 @@ class HTTPService(Connector):
     name: Optional[str] = None
     health_endpoint: Optional[str] = "health"
 
-    def __init__(self, host: str, source: str, timeout: int = 5, retries: int = 5):
+    def __init__(
+        self,
+        host: str,
+        source: str,
+        timeout: int = 5,
+        retries: int = 5,
+    ):
```

Comment:
If I see yet another change that preaches to set black's line-length to 80, I will escalate this to our code style officer.

Reply:
argh, busted... Hey, I'm even doing what black is suggesting and get no warnings from pre-commit :p